[2021-06-15 09:05:36] Request: 
open_dataset(ds_id=esacci.AEROSOL.day.L3C.AOD.MERIS.Envisat.MERIS_ENVISAT.2-2.r1, store_id=cci-store, time_range=None, var_names=['AOD550', 'AOD550_std'], region=None)

Traceback (most recent call last):
  File "/home/tonio-bc/projects/ccitools/cate-e2e/testing-cci-datasets/test_cci_data_support.py", line 354, in test_open_ds
    force_local=False)
  File "/home/tonio-bc/projects/ccitools/cate/cate/core/op.py", line 216, in __call__
    return_value = self._wrapped_op(**input_values)
  File "/home/tonio-bc/projects/ccitools/cate/cate/ops/io.py", line 92, in open_dataset
    monitor=monitor)
  File "/home/tonio-bc/projects/ccitools/cate/cate/core/ds.py", line 456, in open_dataset
    dataset = data_store.open_data(data_id=dataset_id, opener_id=opener_id, **open_args)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/dataaccess.py", line 472, in open_data
    return self._get_opener(opener_id=opener_id).open_data(data_id, **open_params)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/dataaccess.py", line 245, in open_data
    chunk_store = CciChunkStore(self._cci_odp, data_id, cube_kwargs)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/chunkstore.py", line 664, in __init__
    trace_store_calls=trace_store_calls)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/chunkstore.py", line 197, in __init__
    sizes.append(self._dimension_data[dimension_name]['size'])
KeyError: 'latitude'
[2021-06-15 09:11:32] Request: 
open_dataset(ds_id=esacci.AEROSOL.day.L3C.AOD.MERIS.Envisat.MERIS_ENVISAT.2-2.r1, store_id=cci-store, time_range=None, var_names=['AOD865_std', 'AOD550_std'], region=None)

Traceback (most recent call last):
  File "/home/tonio-bc/projects/ccitools/cate-e2e/testing-cci-datasets/test_cci_data_support.py", line 354, in test_open_ds
    force_local=False)
  File "/home/tonio-bc/projects/ccitools/cate/cate/core/op.py", line 216, in __call__
    return_value = self._wrapped_op(**input_values)
  File "/home/tonio-bc/projects/ccitools/cate/cate/ops/io.py", line 92, in open_dataset
    monitor=monitor)
  File "/home/tonio-bc/projects/ccitools/cate/cate/core/ds.py", line 456, in open_dataset
    dataset = data_store.open_data(data_id=dataset_id, opener_id=opener_id, **open_args)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/dataaccess.py", line 472, in open_data
    return self._get_opener(opener_id=opener_id).open_data(data_id, **open_params)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/dataaccess.py", line 245, in open_data
    chunk_store = CciChunkStore(self._cci_odp, data_id, cube_kwargs)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/chunkstore.py", line 664, in __init__
    trace_store_calls=trace_store_calls)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/chunkstore.py", line 197, in __init__
    sizes.append(self._dimension_data[dimension_name]['size'])
KeyError: 'latitude'
[2021-06-15 09:27:57] Request: 
open_dataset(ds_id=esacci.AEROSOL.day.L3C.AOD.MERIS.Envisat.MERIS_ENVISAT.2-2.r1, store_id=cci-store, time_range=None, var_names=['fAOD865_std', 'fAOD865'], region=None)

Traceback (most recent call last):
  File "/home/tonio-bc/projects/ccitools/cate-e2e/testing-cci-datasets/test_cci_data_support.py", line 354, in test_open_ds
    force_local=False)
  File "/home/tonio-bc/projects/ccitools/cate/cate/core/op.py", line 216, in __call__
    return_value = self._wrapped_op(**input_values)
  File "/home/tonio-bc/projects/ccitools/cate/cate/ops/io.py", line 92, in open_dataset
    monitor=monitor)
  File "/home/tonio-bc/projects/ccitools/cate/cate/core/ds.py", line 456, in open_dataset
    dataset = data_store.open_data(data_id=dataset_id, opener_id=opener_id, **open_args)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/dataaccess.py", line 472, in open_data
    return self._get_opener(opener_id=opener_id).open_data(data_id, **open_params)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/dataaccess.py", line 245, in open_data
    chunk_store = CciChunkStore(self._cci_odp, data_id, cube_kwargs)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/chunkstore.py", line 664, in __init__
    trace_store_calls=trace_store_calls)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/chunkstore.py", line 197, in __init__
    sizes.append(self._dimension_data[dimension_name]['size'])
KeyError: 'latitude'
[2021-06-15 09:57:05] Request: 
open_dataset(ds_id=esacci.AEROSOL.day.L3C.AOD.MERIS.Envisat.MERIS_ENVISAT.2-2.r1, store_id=cci-store, time_range=None, var_names=['AOD865', 'AOD550_std'], region=None)

Traceback (most recent call last):
  File "/home/tonio-bc/projects/ccitools/cate-e2e/testing-cci-datasets/test_cci_data_support.py", line 354, in test_open_ds
    force_local=False)
  File "/home/tonio-bc/projects/ccitools/cate/cate/core/op.py", line 216, in __call__
    return_value = self._wrapped_op(**input_values)
  File "/home/tonio-bc/projects/ccitools/cate/cate/ops/io.py", line 92, in open_dataset
    monitor=monitor)
  File "/home/tonio-bc/projects/ccitools/cate/cate/core/ds.py", line 456, in open_dataset
    dataset = data_store.open_data(data_id=dataset_id, opener_id=opener_id, **open_args)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/dataaccess.py", line 472, in open_data
    return self._get_opener(opener_id=opener_id).open_data(data_id, **open_params)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/dataaccess.py", line 245, in open_data
    chunk_store = CciChunkStore(self._cci_odp, data_id, cube_kwargs)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/chunkstore.py", line 664, in __init__
    trace_store_calls=trace_store_calls)
  File "/home/tonio-bc/projects/dcs4cop/xcube-cci/xcube_cci/chunkstore.py", line 197, in __init__
    sizes.append(self._dimension_data[dimension_name]['size'])
KeyError: 'latitude'
